soccer/planning/Path.hpp

```diff
@@ -18,6 +25,9 @@ class Path {
     Path() : _startTime(0) {}
     virtual ~Path() {}
 
+    /// Returns true if the path is non-empty
+    virtual bool valid() const { return true; }
```

Comment:
That's fair.  Every path planner should return a valid path, so we shouldn't have to check elsewhere if it's valid.  I'll remove the `valid()`.